classes/item.rb

```diff
@@ -0,0 +1,42 @@
+require 'date'
+
+class Item
+  attr_reader :genre, :author, :source, :label
+
+  # rubocop: disable all
+  def initialize(id, publish_date, archived = false)
```

Comment:
- [ ] Why are you disabling the rubocop errors, you included the rubocop.yml file to help you to deal with and correct the rubocop errors however you are disabling all errors. Try to correct some errors if it doesn't fix all you can mention it.

Reply:
Sorry Sir, There is a rubocop conflict when using that syntax to parse the arguments. 

If I use this syntax 

```cmd
 archived = false 
```

The code works but I got a rubocop error and it suggests I use the following way :

```cmd
archived: false
``` 
Then I don't get rubocop error but the code doesn't work and complains about the number of arguments being not 3.